torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -235,15 +235,38 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
     updated_val = at::add(inputTensorValues[0], inputTensorValues[1]);
     return c10::optional<at::Tensor>(updated_val);
   } else if (node->kind() == onnx::Unsqueeze) {
-    assert(inputTensorValues.size() == 1);
-    if (!node->hasAttributeS("axes")) {
+    if (opset_version == ONNX_OPSET_13){
+      assert(inputTensorValues.size() == 2);
+      // Checking validity of 'axes' input
+      if (inputTensorValues[1].sizes().size() != 1) {
+        std::cerr
+            << "Warning: Constant folding - Invalid 'axes' inputs found for opset 13 onnx::Unsqueeze op. "
+            << "Constant folding not applied." << std::endl;
+        return c10::nullopt;
+      }
+      auto axes = inputTensorValues[1].accessor<int64_t, 1>();
+      updated_val = inputTensorValues[0];
+      for (int64_t i = 0; i < inputTensorValues[1].sizes()[0]; ++i) {
```

Comment:
There will be issue for edge cases if there are multiple `axes`, and the order of `axes` affects the result. Try in pytorch 

```
import torch
a = torch.ones(3, 4)
a.unsqueeze(0).unsqueeze(2).shape
a.unsqueeze(2).unsqueeze(0).shape
```

swapping the order produces different result.

Reply:
Thanks. will consider to support __axes__ negative numbers and sort them to fix this edge cases.

Reply:
Thanks. Please add these cases in utility tests.